src/backend/UnaryOperator.js

```diff
@@ -9,15 +9,23 @@ const Node = require('./Node.js');
 
 class UnaryOperator extends Node {
 
-  constructor() {
+  constructor(operatorText) {
```

Comment:
Same thing as the other node class your wrote. Defaults might be a nice way to still allow entering if needed for flexibility